internal/driver/glfw/driver.go

```diff
@@ -71,6 +75,17 @@ func (d *gLDriver) Run() {
 	d.runGL()
 }
 
+func (d *gLDriver) StartAnimation(a *fyne.Animation) {
+	d.animationMutex.Lock()
+	defer d.animationMutex.Unlock()
+	wasStopped := len(d.animations) == 0
+
+	d.animations = append(d.animations, &anim{a, time.Now(), time.Now().Add(a.Duration)})
+	if wasStopped {
```

Comment:
Could save a var & line here :)
```suggestion
	if len(d.animations) == 1 {
```

Reply:
True. I just implemented this then felt the need to write "// was Stopped" at the end, so I'm not convinced it's clearer.
The `== 0` is clearer and the variable name marks it's significance in my opinion.